index.html

```diff
@@ -356,11 +348,10 @@
       <p>HTTP/2 ([[!RFC7540]]) allows a server to pre-emptively send ("push")
       responses to the client. A pushed response is semantically equivalent to
       a server responding to a request and, similar to a preloaded response, is
-      retained by the user agent and executed by the application when
-      <a data-lt="match a preloaded response">matched with another request</a>
-      initiated by the application. As such, from an application perspective,
-      there is no difference between consuming a preload or a server push
-      response.</p>
+      retained by the user agent and executed by the application when matched
+      with a request initiated by the application. As such, from an application
+      perspective, there is no difference between consuming a preload or a
+      server push response.</p>
```

Comment:
That's not entirely true. There's no difference between consuming an explicit preload (link preload based) and an implicit one (triggered by preloadScanner). There is a difference with H2 push though - consuming H2 push involves marking an RT entry, going through SW, and querying the HTTP cache before the H2 push response is matches with the request at the net stack layer.